src/table/table.scss

```diff
@@ -122,265 +249,341 @@
-    justify-content: center;
-    height: 100%;
-    padding: $iui-xl;
-  }
+        *::selection {
+          @include themed {
+            background-color: rgba(
+              t(iui-color-foreground-warning-rgb),
+              t(iui-opacity-4)
+            );
+          }
+        }
+      }
 
-  .iui-tables-paginator {
-    @include themed {
-      background-color: t(iui-color-background-3);
+      .iui-status {
+        @include themed {
+          fill: t(iui-icons-color-warning);
+        }
+      }
     }
-    user-select: none;
 
-    &:hover {
-      @include themed {
-        background-color: t(iui-color-background-3) !important;
+    &.iui-negative {
+      &,
+      + .iui-expanded-content {
+        @include themed {
+          box-shadow: inset $iui-xxs 0 0 0 t(iui-icons-color-negative);
+        }
+
+        *::selection {
+          @include themed {
+            background-color: rgba(
+              t(iui-color-foreground-negative-rgb),
+              t(iui-opacity-4)
+            );
+          }
+        }
       }
-    }
 
-    > .iui-content {
-      @include themed {
-        border-bottom: solid 1px t(iui-color-background-4);
+      .iui-status {
+        @include themed {
+          fill: t(iui-icons-color-negative);
+        }
       }
-      width: 100%;
-      display: flex;
-      justify-content: space-around;
-      align-items: center;
-      padding: $iui-table-density;
-      flex: 1;
+    }
 
-      > .iui-full-line {
-        flex: 1;
+    &.iui-expanded {
+      .iui-expander > .iui-icon {
+        transform: rotate(90deg);
       }
 
-      > .iui-controls {
+      + .iui-expanded-content {
         @include themed {
-          color: t(iui-text-color);
+          border-top: solid 1px t(iui-color-background-1);
         }
-        width: 300px;
-        display: flex;
-        justify-content: center;
-        align-items: center;
-        flex: 1;
+      }
+    }
+    // #endregion Selection
+  }
 
-        > a {
-          @include iui-anchor;
-          margin: 0 $iui-s;
-        }
+  // Empty & loading states
+  &.iui-message {
+    text-align: center;
+    height: 100%;
+    padding: $iui-xl;
+    @include themed {
+      color: t(iui-text-color-muted);
+      background-color: t(iui-color-background-2);
+    }
+  }
+}
 
-        > .iui-ellipsis {
-          margin: 0 $iui-s;
-        }
+@mixin iui-table-cell {
+  align-items: center;
+  display: flex;
+  flex-grow: 1;
+  min-width: $iui-l * 6;
+  min-height: $iui-baseline * 5;
+  padding: 0 $iui-m;
+  overflow: hidden;
+  list-style: none;
+  flex-basis: $iui-l * 6;
+
+  > svg {
+    width: $iui-icons-default;
+    height: $iui-icons-default;
+    flex-shrink: 0;
+  }
 
-        > .iui-current,
-        > .iui-ellipsis {
-          @include themed {
-            color: t(iui-text-color);
-          }
-          cursor: default;
-        }
+  > .iui-status,
+  > .iui-progress-indicator-radial {
+    float: right;
+    margin-left: auto;
+  }
```

Comment:
This also seems a bit random

Reply:
This was part of the special cell rules which do look random. Honestly, I'm not sure if this should be in table.scss or in backstop/table.html saying it's custom style. At this point, it's work and integration look good, so I wouldn't remove that, but this is custom cell component and not table's cell.

Reply:
Not sure float has any influence at all...
I think we should not have such specific rules and think about the ways of generalising the placement of elements in the Cell versus tackling all different cases (its endless)
We need a separate discussion about the approach we could take here.. We will end up in lots of bugs if we try to tackle all possible separate components in the cell..

Reply:
and again..duplication, duplication, duplication..
![image](https://user-images.githubusercontent.com/81580355/120781711-fb975380-c531-11eb-812c-65f6260f7fba.png)

Reply:
Float has been removed.  I've changed the CSS to no longer target the SVG or progress indicator directly, instead it now targets the `div.iui-status` & you place the SVG or progress indicator inside of it.